src/+matlode/+startingstep/SetStep.m

```diff
@@ -0,0 +1,31 @@
+classdef SetStep < matlode.startingstep.StartingStep
+    %SETSTEP Summary of this class goes here
+    %   Detailed explanation goes here
+    
+    properties
+        StartingStep
+    end
+    
+    methods
+        function obj = SetStep(ssp, varargin)
+            obj = obj@matlode.startingstep.StartingStep(varargin{:});
+            obj.StartingStep = ssp;
+        end
+        
+        function [h0, f0, fevals] = startingStep(obj, ~, tspan, ~, ~, ~, minStep, maxStep)
+            fevals = 0;
+            f0 = [];
+            if obj.StartingStep > maxStep
+               error('The starting step is greater then the max step') 
+            end
+            if obj.StartingStep < minStep
+                error('The starting step cannot be so small')
```

Comment:
So small as what? explicitly state it in the error message. Recall that you can treat error as an fprintf